docs/api.md

```diff
@@ -285,6 +285,108 @@ GET https://formulae.brew.sh/api/formula-linux/${FORMULA}.json
 }
 ```
 
+### Get bottle metadata for a {{ site.taps.core.repo }} formula
+Get the `brew info --json --bottle --formula <formula>` output for a single, current {{ site.taps.core.fullname }} or {{ site.taps.linux.fullname }} formula.
+
+```
+GET https://formulae.brew.sh/api/bottle/${FORMULA}.json
+GET https://formulae.brew.sh/api/bottle-linux/${FORMULA}.json
+```
+
+#### Variables
+- `${FORMULA}`: the name of the formula, e.g. `wget`
+
+#### [Response](https://formulae.brew.sh/api/bottle/wget.json)
+```json
+{
+  "name": "wget",
+  "bottles": {
+    "arm64_big_sur": {
+      "url": "https://ghcr.io/v2/homebrew/core/wget/blobs/sha256:e9034fc9062d5d28972135be031876672aff18fa945ce37e9c2ee1e2c4287f3a"
+    },
```

Comment:
```suggestion
    "arm64_big_sur": "https://ghcr.io/v2/homebrew/core/wget/blobs/sha256:e9034fc9062d5d28972135be031876672aff18fa945ce37e9c2ee1e2c4287f3a",
```

I almost wonder if this syntax would be a nicer shorthand for when the `sha256` isn't specified? There's going to need to be a different parse path for handling the missing `sha256` case anyway?

Reply:
Or even:

```suggestion
    "arm64_big_sur": "$BLOBS/e9034fc9062d5d28972135be031876672aff18fa945ce37e9c2ee1e2c4287f3a",
```

given that the string prefix is identical every time.